azurerm/resource_arm_application_gateway_test.go

```diff
@@ -1939,9 +2005,259 @@ resource "azurerm_application_gateway" "test" {
     capacity = 1
   }
 
-  disabled_ssl_protocols = [
-    "TLSv1_0",
-  ]
```

Comment:
can we add a test covering the existing behaviour (this ^) to ensure it still works for users?